client/gameObjects/Skeleton.js

```diff
@@ -282,6 +315,13 @@ export default class Skeleton extends Phaser.GameObjects.Image {
       } else {
         this.hideRange();
       }
+    } else if (!this.isMainPlayer) {
```

Comment:
Do we need 'else if'? Else is not enough?

Reply:
It can be either this or `else (this.isDead) {` (because of ESLint - which only makes it more chaotic for me, which one do you prefer?